indexing.rdf

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="rdfs2html.xsl"?>
+<rdf:RDF 
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:dcterms="http://purl.org/dc/terms/"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+
+    <rdf:Description rdf:about="http://fedora.info/definitions/v4/indexing#">
+      <dcterms:title xml:lang="en">Fedora Commons Indexing Ontology</dcterms:title>
+      <dcterms:modified rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2015-01-23</dcterms:modified>
+      <dcterms:publisher rdf:resource="http://fedora.info"/>
```

Comment:
All that is fine with me. For the future, we might want to think about dedicating one of our many domains to "spec stuff", and keeping the others for stuff relating to our reference impl.